service/models/wishlist.py

```diff
@@ -11,24 +11,141 @@
 
 """
 
-class Wishlist():
-  def create():
-    pass
+from flask import Flask
+from sqlalchemy.orm import query
+from service.models.product import Product
+from service.models.wishlist_product import WishlistProduct
+from .model_utils import EntityNotFoundError, db,logger,Availability,DataValidationError
+
+class Wishlist(db.Model):
+  __tablename__ = 'wishlist'
+
+  app:Flask = None
+
+  id = db.Column(db.Integer,primary_key = True)
+  name = db.Column(db.String(64), nullable=False)
+  user_id = db.Column(db.Integer, nullable=False)
```

Comment:
Another attribute we had discussed is creation timestamp - I think we should add it.